packages/documentation-framework/components/tableOfContents/tableOfContents.css

```
@@ -2,8 +2,8 @@
   align-self: flex-start;
   position: sticky;
   width: calc(100% + var(--pf-v6-c-page__main-section--PaddingLeft) + var(--pf-v6-c-page__main-section--PaddingRight));
-  background-color: var(--pf-t--global--background--color--primary--default);
-  z-index: 501;
+  background-color: var(--pf-t--global--background--color--200);
```

I haven't really dug into the styles too much, but looks like since this applies to `.ws-toc`, you shouldn't need to specify a different background on the jumplinks header here https://github.com/patternfly/patternfly-org/blob/27938f190eff1be7b8fdd9b82b256454155047fe/packages/documentation-framework/components/tableOfContents/tableOfContents.css#L32

Though whether you keep it or not, this background on `.ws-toc` should use the semantic var unless there's a reason you don't think it's the right var?

```suggestion
  background-color: var(--pf-t--global--background--color--secondary--default);
```